templates/deno/src/client.ts.twig

```diff
@@ -85,6 +85,9 @@ export class Client {
 
             body = formData;
         } else {
+            if ('queries' in params && Array.isArray(params['queries'])) {
+                params['queries'] = params['queries'].map((query) => query.toString());
+            }
```

Comment:
We need to do this in the GET case, routes with queries will always be GET

Reply:
Done inside flatten() function instead.